server/spec/helpers/wait_helper_spec.rb

```diff
@@ -18,24 +18,34 @@
   end
 
   describe 'wait' do
-    it 'returns true immediately' do
+    it 'returns true immediately without any logging' do
       expect(subject).not_to receive(:sleep)
+      expect(subject).not_to receive(:debug)
 
       value = subject.wait_until { true }
 
       expect(value).to be_truthy
     end
 
-    it 'sleeps between retries and logs debug' do
-      expect(subject).to receive(:debug).with('waiting 0.5s of 2.0s until: something that is true the second time')
-      expect(subject).to receive(:debug).with('waited 0.5s until: something that is true the second time yielded true')
+    it 'sleeps between retries and logs debug while under threshold' do
```

Comment:
It already mocks out the `sleep` and keeps an mock clock, so these specs shouldn't be slowed.